pages/utils/sum.ts

```diff
@@ -0,0 +1,8 @@
+export function sum(a: number, b: number, format: "html" | "text" = "html") {
```

Comment:
Rename parameter to "num1" or "num2" to make it more readable compared to a and b maybe

Reply:
Yes. I agree. Lets do that.